tasks/publish.yaml

```
@@ -5,63 +5,48 @@ variables:
   - name: FLAVOR
     default: upstream
 
+  - name: UDS_ARCH
+
   - name: VERSION
     description: "The version of the packages to build"
     # x-release-please-start-version
     default: "0.13.1"
     # x-release-please-end
 
 tasks:
-  - name: packages
+  - name: standard package
```

I think these names all need to be hyphenated? Expect this would throw a syntax issue/something about the space when called, and the workflows are calling with a hyphen currently.
```suggestion
  - name: standard-package
```